### PR TITLE
Generic JSON order file to Odoo

### PR DESCRIPTION
**Pull request by FrendsSarlinS**
#69 

## Summary by CodeRabbit

- **New Features**
  - Introduced a template for retrieving, mapping, and transferring order files from an SFTP server to Odoo.
  - Added metadata information for the "Generic JSON order File to Odoo" functionality.
  - Defined a process for importing sales orders into Odoo from an SFTP server.

### CODE REVIEW

File: Templates/CRMs/Odoo/Generic JSON order file to Odoo/long-description.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+This template assumes that the following prerequisites are in place:
+
+- SFTP address, username and password are required to access the file.
+- Odoo user exists for performing the synchronization.
+- Odoo has a database with usable products, partners, addresses and a pricelist for the synchronization.
+
+# Implementation and Usage Notes
+
+This template is used to insert order data to Odoo using Odoo models for order and order line. Other properties, such as activation, deactivation and deletion is not implemented in this template.
+Process variables include fields for both SFTP and Odoo credentials, SFTP and Odoo addresses, SFTP file path, SFTP file name and Odoo database name.
```
Comment (jefim) [inline Templates/CRMs/Odoo/Generic JSON order file to Odoo/long-description.md:17]: Please include a JSON file example that this template is able to handle.

**Review by jefim — CHANGES_REQUESTED** [review]

File: Templates/CRMs/Odoo/Generic JSON order file to Odoo/long-description.md (hunk 1 above)
Comment (FrendsSarlinS) [reply Templates/CRMs/Odoo/Generic JSON order file to Odoo/long-description.md:17]: Example data added!

File: Templates/CRMs/Odoo/Generic JSON order file to Odoo/long-description.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+This template assumes that the following prerequisites are in place:
+
+- SFTP address, username and password are required to access the file.
+- Odoo user exists for performing the synchronization.
+- Odoo has a database with usable products, partners, addresses and a pricelist for the synchronization.
+
+# Implementation and Usage Notes
+
+This template is used to insert order data to Odoo using Odoo models for order and order line. Other properties, such as activation, deactivation and deletion is not implemented in this template.
```
[the remark this answers is not in the document]
Comment (jefim) [reply Templates/CRMs/Odoo/Generic JSON order file to Odoo/long-description.md:16]: This is a good suggestion :)

File: Templates/CRMs/Odoo/Generic JSON order file to Odoo/long-description.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+          "state": "sale"
+      }
+  ]
+}
+
+```
+
+# Error Handling
+
+There are error checks after the tasks for reading the SFTP file and inserting Odoo data. If a task is not successful, error message will be appended to an exception variable and displayed at the end of the process. Next order or order line is handled after an error.
```
[the remark this answers is not in the document]
Comment (jefim) [reply Templates/CRMs/Odoo/Generic JSON order file to Odoo/long-description.md:63]: This is also a good suggestion

**Review by jefim — APPROVED** [review]

**Pull request merged**